wpilib/wpilib/sensorbase.py

```diff
@@ -163,6 +163,9 @@ def getDefaultSolenoidModule():
         """
         return SensorBase.defaultSolenoidModule
 
+    def __init__(self):
+        super().__init__()
+
```

Comment:
Why have you added this useless constructor?

Reply:
SendableBase.__init__ must be called

Reply:
Strictly speaking, if you omit the constructor, then super will be called instead. But, that's fine.